templates/repo/create.tmpl

```diff
@@ -170,19 +170,20 @@
 						<div class="inline field">
 							<label>{{.i18n.Tr "repo.settings.trust_model"}}</label>
 							<div class="ui selection owner dropdown">
-							<input type="hidden" id="trust_model" name="trust_model" value="default" required>
-							<div class="default text">{{.i18n.Tr "repo.settings.trust_model"}}</div>
-							<i class="dropdown icon"></i>
-							<div class="menu">
-								<div class="item" data-value="default">{{.i18n.Tr "repo.settings.trust_model.default"}}</div>
-								<div class="item" data-value="collaborator">{{.i18n.Tr "repo.settings.trust_model.collaborator"}}</div>
-								<div class="item" data-value="committer">{{.i18n.Tr "repo.settings.trust_model.committer"}}</div>
-								<div class="item" data-value="collaboratorcommitter">{{.i18n.Tr "repo.settings.trust_model.collaboratorcommitter"}}</div>
+								<input type="hidden" id="trust_model" name="trust_model" value="default" required>
+								<div class="default text">{{.i18n.Tr "repo.settings.trust_model"}}</div>
+								<i class="dropdown icon"></i>
+								<div class="menu">
+									<div class="item" data-value="default">{{.i18n.Tr "repo.settings.trust_model.default"}}</div>
+									<div class="item" data-value="collaborator">{{.i18n.Tr "repo.settings.trust_model.collaborator"}}</div>
+									<div class="item" data-value="committer">{{.i18n.Tr "repo.settings.trust_model.committer"}}</div>
+									<div class="item" data-value="collaboratorcommitter">{{.i18n.Tr "repo.settings.trust_model.collaboratorcommitter"}}</div>
+								</div>
 							</div>
 						</div>
 					</div>
 
-					<br/>
+					<br />
```

Comment:
A quick look through templates suggests that we currently use two variants:
- `<br>`
- `<br/>`

Why is this PR introducing yet another one?